plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployCommandHandler.java

```diff
@@ -208,8 +192,7 @@ private static void launchCleanupJob() {
   }
 
   private static IPath getTempDir() {
-    return Platform
-        .getStateLocation(Platform.getBundle("com.google.cloud.tools.eclipse.appengine.deploy"))
+    return Platform.getStateLocation(FrameworkUtil.getBundle(DeployJob.class))
```

Comment:
I noticed that previously it isn't the `.deploy.ui` but just `.deploy'. I think the first author wanted to have the directory under the non-UI bundle, which seems OK to me.